### PR TITLE
Modifies `docker network create`

### LINKED ISSUE #1481: Improvements to "network create" reference
File: [engine/reference/commandline/network_create.md](https://docs.docker.com/engine/reference/commandline/network_create/)

1. The options of `docker create network` are listed, but not documented. Especially the arguments to the options and their requirements and constraints are not completely described.
2. The structure of the page is confusing: It starts with some general stuff, goes into great detail for the overlay driver, and briefly returns to the bridge driver again.
3. The section "specifying advanced options" is verbatimly duplicated in the section "Examples".
4. The example with `--aux-address` is at least misleading, the mentioned option never shows up in the actual example. In no part of this page it is explained what `--aux-address` is actual for.
5. I am looking for some information about how the engine ties the *host* to these newly created networks, especially if I have more than a single interface on my host.

### PR DESCRIPTION
**Pull request by surajnarwade**
Resolves #1481

### CODE REVIEW

File: _data/engine-cli-edge/docker_network_create.yaml [hunk 1]
```diff
@@ -112,6 +116,10 @@ examples: |-
   You can disconnect a container from a network using the `docker network
   disconnect` command.
 
+  ``bash
+      $ docker network disconnect mynet mycontainer
+      ```
```
Comment (johndmulhausen) [inline _data/engine-cli-edge/docker_network_create.yaml]: Oops, looks like things got misaligned here.

File: _data/engine-cli/docker_network_create.yaml [hunk 2]
```diff
@@ -103,6 +103,10 @@ examples: |-
   If you want to add a container to a network after the container is already
   running, use the `docker network connect` subcommand.
 
+  ``bash
+      $ docker network connect mynet mycontainer
+      ```
```
Comment (johndmulhausen) [inline _data/engine-cli/docker_network_create.yaml]: Misaligned

File: _data/engine-cli/docker_network_create.yaml [hunk 3]
```diff
@@ -112,6 +116,10 @@ examples: |-
   You can disconnect a container from a network using the `docker network
   disconnect` command.
 
+   ```bash
+    $ docker network disconnect mynet mycontainer
+    ```
```
Comment (johndmulhausen) [inline _data/engine-cli/docker_network_create.yaml]: Misaligned.

**Comment by surajnarwade** [comment]
@mstanleyjones I am onto that now

**Pull request closed without merging**